features/login.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Smoke
+
+  Scenario Outline: Change Currency on site
+    Given I open the home page as a registered user
+    When I click to currency button
```

Comment:
In Gherkin, we also have the "And" step that is better to use instead of multiplying "When"